modules/frontend/combiner/metrics_query_range.go

```diff
@@ -60,6 +61,7 @@ func NewTypedQueryRange(req *tempopb.QueryRangeRequest, trackDiffs bool) (GRPCCo
 		return nil, err
 	}
 
+	initHTTPCombiner(c.(*genericCombiner[*tempopb.QueryRangeResponse]), api.HeaderAcceptJSON)
```

Comment:
is there a reason not to do this in the `NewQueryRange` func above? 

same Q for all below.

Reply:
Right I got confused with the typed methods